src/services/videomania/index.jsx

```
@@ -248,3 +255,205 @@ export const AddVideoMania = ({
+  const [commentText, setCommentText] = useState("");
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const handleComment = async () => {
+    if (!commentText.trim()) return;
+
+    const previousComments = [...comments];
+
+    try {
+      const payload = {
+        user_id: user?.id,
+        video_id: video?.video_id,
```

ESLint

[error] 'video.video_id' is missing in props validation

(react/prop-types)